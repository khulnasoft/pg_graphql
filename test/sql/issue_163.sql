begin;
    create table profiles(
        id int primary key,
        username text
    );

    insert into public.profiles(id, username)
    values
        (1, 'foo');

    select jsonb_pretty(
        graphql.resolve($$
            query MyQuery {
              __typename
              profilesCollection {
                edges {
                  node {
                    id
                    username
                  }
                }
              }
            }
        $$)
    )

rollback;
