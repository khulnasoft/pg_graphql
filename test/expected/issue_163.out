begin;
    create table profiles(
        id int primary key,
        username text
    );
    insert into public.profiles(id, username)
    values
        (1, 'foo');
    select jsonb_pretty(
        graphql.resolve($$
            query MyQuery {
              __typename
              profilesCollection {
                edges {
                  node {
                    id
                    username
                  }
                }
              }
            }
        $$)
    )
rollback;
                 rollback                  
-------------------------------------------
 {                                        +
     "data": {                            +
         "__typename": "Query",           +
         "profilesCollection": {          +
             "edges": [                   +
                 {                        +
                     "node": {            +
                         "id": 1,         +
                         "username": "foo"+
                     }                    +
                 }                        +
             ]                            +
         }                                +
     }                                    +
 }
(1 row)

